keycloak/themes/hmda/login/theme.properties

```diff
@@ -1,5 +1,5 @@
 parent=keycloak
 scripts=lib/jquery/jquery-1.10.2.js lib/select2-3.4.1/select2.min.js
-styles=lib/patternfly/css/patternfly.css lib/zocial/zocial.css lib/select2-3.4.1/select2.css css/login.css
+styles=css/uswds.min.css lib/select2-3.4.1/select2.css css/select2-custom.css
 
-institutionSearchUri={{INSTITUTION_SEARCH_URI}}
```

Comment:
Yeah, sorry. Didn't commit that before but must have slipped through at some point.